src/AEOS/AEOSConnector/Factories/AEOSConnectorFactory.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+
+using Innovatrics.SmartFace.Integrations.AEOSConnector.Connectors;
+
+namespace Innovatrics.SmartFace.Integrations.AEOSConnector.Factories
+{
+    public class AEOSConnectorFactory : IAEOSConnectorFactory
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+        private readonly IHttpClientFactory httpClientFactory;
+
+        public AEOSConnectorFactory(
+            ILogger logger,
+            IConfiguration configuration,
+            IHttpClientFactory httpClientFactory
+        )
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+            this.httpClientFactory = httpClientFactory ?? throw new ArgumentNullException(nameof(httpClientFactory));
+        }
+
+        public IAEOSConnector Create(string type)
+        {
+            if (type == null)
+            {
+                throw new ArgumentNullException(nameof(type));
+            }
+
+            this.logger.Information("Creating IAEOSConnector for type {type}", type);
+
+            type = type
+                    .ReplaceAll(new string[] { "-", " ", "." }, new string[] { "_", "_", "_" })
+                    .ToUpper();
+
+            //this.logger.Information("type: {type}", type);
+            switch (type)
+            {
+                default:
+                    throw new NotImplementedException($"AEOS Connector of type {type} not supported");
+
+                case "AEPU":
+                    return new AEpuConnector(this.logger, this.configuration, this.httpClientFactory);
```

Comment:
this should be rethinked.. Are we connecting to AEOS or AEPU?

IF AEPU, then rename whole solution AEOS => AEPU
IF AEOS, then remove AEPU

..or argument back why it is so

Reply:
the AEOSConnector was renamed to be AEpuConnector as it connects to AEpu controller within the NEDAP'S AEOS access control solution. The AEOSSyncTool is renamed to AeosSync and refactored across the project.